MonstieWash/Assets/Scripts/Erasing/Tool.cs

```diff
@@ -6,18 +6,22 @@
 [CreateAssetMenu(fileName = "Tool", menuName = "ScriptableObjects/Tool")]
 public class Tool : ScriptableObject
 {
+    public enum ToolType { Brush, None, Sponge, WaterWand };
```

Comment:
**Suggestion** | _This is a suggestion, feel free to take it or leave it._

I think it makes more sense for this to be ordered with 'None' in the first spot.

```suggestion
    public enum ToolType { None, Brush, Sponge, WaterWand };
```